### PR TITLE
Update ODAG generation actions to work with June 2020

### PR DESCRIPTION
**Pull request by SeBBBe**
**Checklist**

- [ ] tests added
- [x] commits conform to the [Commit message format](./CONTRIBUTING.md#commit)
- [ ] documentation updated

**Squash commit message**

Update ODAG generation actions to work with June 2020

### CODE REVIEW

File: senseobjects/odagtoolbarnavpoint.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	}
+)
+
+func (otn *OdagToolbarNavpoint) setLayout(layout *OdagToolbarNavpointLayout) {
+	otn.mutex.Lock()
+	defer otn.mutex.Unlock()
+	otn.layout = layout
+}
+
+// UpdateLayout get and set a new layout for sheetlist
```
Comment (DnlLrssn) [inline senseobjects/odagtoolbarnavpoint.go:33]: not sheetlist

File: scenario/generateodag.go [hunk 2]
```diff
@@ -372,12 +379,12 @@ func GetOdagSelectionBindings(host string, odagLinkId string, sessionState *sess
...
-// createFieldListboxAsync creates a listbox session object for specified field
-func createFieldListboxAsync(sessionState *session.State, actionState *action.State, doc *enigma.Doc, field string) (*senseobjects.ListBox, error) {
-	var obj *senseobjects.ListBox
+// createOdagToolbarNavpointAsync creates a listbox session object for specified field
+func createOdagToolbarNavpointAsync(sessionState *session.State, actionState *action.State, doc *enigma.Doc, field string) (*senseobjects.OdagToolbarNavpoint, error) {
+	var obj *senseobjects.OdagToolbarNavpoint
 	err := sessionState.SendRequest(actionState, func(ctx context.Context) error {
 		var err error
-		obj, err = senseobjects.CreateListBoxObject(ctx, doc, field)
+		obj, err = senseobjects.CreateOdagToolbarNavpoint(ctx, doc, field)
```
Comment (DnlLrssn) [inline scenario/generateodag.go:387]: Is a new session object created for every ODAG request in client? E.g. considering this scenario
1. selects x,y
2. Create ODAG
3. select v,w
4. Create ODAG
Should 2 session objects be created or is the object in (2) updated through normal update mechanism and updated during (3)?

Comment (SeBBBe) [reply scenario/generateodag.go:387]: They are always created (and instantly destroyed after the request has been sent). So it follows the client flow.

File: senseobjects/odagtoolbarnavpoint.go (hunk 1 above)
Comment (SeBBBe) [reply senseobjects/odagtoolbarnavpoint.go:33]: Will fix

File: scenario/generateodag.go (hunk 2 above)
Comment (DnlLrssn) [reply scenario/generateodag.go:387]: ah I see when looking in full code, didn't show in diff, makes sense then

**Pull request merged**